zigzag/zigzag_test_log.py

```diff
@@ -529,6 +542,17 @@ def _lookup_ids(self):
         except IndexError:  # test case has not been created yet in qTest
             pass
 
+    def _lookup_job_config_attributes(self):
+        """ Finds the array of job config attributes on the end of the job name.
+
+        Returns:
+            List: of job config attributes.
+        """
+        full_name = self._testcase_xml.items()[3][1]
+        under_delimited_list = full_name[full_name.find("[")+1:full_name.find("]")]
+        job_config_attribute_list = under_delimited_list.split("_")
+        return job_config_attribute_list[1:]
```

Comment:
What is the reason for removing the first item from this list?

Reply:
There's a leading underscore, so splitting the string on '_' yields a list with an empty string at the first position.